charts/osm/templates/azure-policy-osm-label.yml

```diff
@@ -0,0 +1,123 @@
+# 
+# The job in this YAML file tries to add 'openservicemesh.io/ignore' label to 'kube-system' and 'azure-arc' namespaces.
```

Comment:
Should `arc-osm-system` also be included in the list of namespaces having this label ?

Reply:
This pre-install hook will only take care of existing namespaces in the cluster before deploying the rest of the resources in this chart. To ensure the namespace containing osm resources (ie. `arc-osm-system`) has the exclusion labels, that should be taken care of by 1 and 2 in this [comment](https://github.com/openservicemesh/osm/issues/1817#issuecomment-707853626).